EmailNotifications/TableStructure.cs

```diff
@@ -70,16 +70,41 @@ public void Dispose()
                 {
                     Append("</tr>");
                 }
-                public void AddCell(string innerText, string classAttributes = "", string id = "", string colSpan = "",string align ="", string  style= "", string fontSize = "")
+                public void AddCell(string innerText, string classAttributes = "", string id = "", string colSpan = "",string align ="", string  style= "", string fontSize = "",bool url = false)
                 {
                     Append("<td");
                     AddOptionalAttributes(classAttributes, id, colSpan, align, style); //put style value in here
+                    if (url == false)
+                    {
                     Append("<font");
-                    AppendOptionFont(classAttributes, id, fontSize,align);
+                    AppendOptionFont(classAttributes, id, fontSize, align);
                     Append(innerText);
                     Append("</font>");
+
+                    }
+                    else
+                    {
+                        Append("<a href="+innerText+">");
+                        Append("<font");
+                        AppendOptionFont(classAttributes, id, fontSize, align);
+                        Append("Here");
+                        Append("</font>");
+
+                        Append("</a>");
+
+                }
+                   
                     Append("</td>");
                 }
+                public void AddImage(string source, string classAttributes = "", string style = "", string id = "", string align = "", string sizeX="",string sizeY = "")
+                {
+                    Append("<td");
```

Comment:
Help me understand. The td element doesn't have this -> ">"

Reply:
follow the methods, its all built in stages, all elements are added eventually